### PR TITLE
added ecct feature flag

### PR DESCRIPTION
**Pull request by gshivach**
feature flag for ECCT start date.
cidev and staging in past for testing purposes.
live in future.

### CODE REVIEW

File: terraform/groups/ecs-service/locals.tf [hunk 1]
```diff
@@ -81,6 +81,7 @@ locals {
     { "name": "FEATURE_FLAG_FIVE_OR_LESS_OFFICERS_JOURNEY_21102021", "value": "${var.feature_flag_five_or_less_officers_journey_21102021}" },
     { "name": "PSC_STATEMENTS_API_PAGE_SIZE", "value": "${var.psc_statements_api_page_size}" },
     { "name": "EWF_URL", "value": "${var.ewf_url}" },
-    { "name": "API_URL", "value": "${var.api_url}" }
+    { "name": "API_URL", "value": "${var.api_url}" },
+    { "name": "FEATURE_FLAG_ECCT_START_DATE_14082023", "value": "${var.feature_flag_ecct_start_date_14082023}"}
```
Comment (mwestacott) [inline terraform/groups/ecs-service/locals.tf:85]: So I'm just wondering why this config item is named as a 'feature flag'? Feature flags are normally are for enabling/disabling functionality and have a true/false value.

Comment (mwestacott) [inline terraform/groups/ecs-service/locals.tf:85]: You also need to add this to the `variables.tf` file too.

Comment (gshivach) [reply terraform/groups/ecs-service/locals.tf:85]: done

File: terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars [hunk 2]
```diff
@@ -19,3 +19,4 @@ feature_flag_active_officers_01072021 = "true"
 feature_flag_five_or_less_officers_journey_21102021 = "true"
 ewf_url = "https://ewf.companieshouse.gov.uk/"
 api_url = "https://private-api.companieshouse.gov.uk"
+feature_flag_ecct_start_date_14082023 = "2024_02-05"
```
Comment (nevans4-ch) [inline terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars:22]: typo here? (_ and - in date)

Comment (gshivach) [reply terraform/groups/ecs-service/profiles/live-eu-west-2/live/vars]: fixed

**Review by nevans4-ch — APPROVED** [review]

**Review by mwestacott — APPROVED** [review]

**Pull request merged**